### ISSUE TITLE
Duplicating in swagger.json file

### ISSUE THREAD

**Issue by azanorth** [Romakita/ts-express-decorators#302]
## Informations

Type | Version
---|---
Question | 4.2.0

****

## Description
I found this duplication when created swagger.json file after reloaded ts-node or rebuild project.
When included reading swagger.json file from the project directory
```typescript
  swagger: {
    path: '/docs',
    specPath: '${rootDir}/swagger.json'
  }
```
In the array of "tags" - tags begin to duplicate constantly after any rebuilding of the project.

Before: 
![tmp2](https://user-images.githubusercontent.com/26149983/37971919-cefd9098-31df-11e8-84b0-9cdc81643bf2.jpg)
It's OK.

After several rebuilding:
![tmp1](https://user-images.githubusercontent.com/26149983/37972144-4c4ea0e6-31e0-11e8-9c94-7e07bcdae736.jpg)
I think that something is wrong here.

Or I making a some mistake. What will you say about this?

**Comment by Romakita** [comment]
If it's to read a swagger.json with default value, it preferable to use `spec` options white a require.

Example:

```typescript
 swagger: {
    path: '/docs',
    spec: require('./swagger.json')
  }
```
try that. I'll change the spec path because his behavior isn't clear and really bugged, my mistake :)

**Comment by Romakita** [comment]
Fixed normally in v4.12.1

**Comment by azanorth** [comment]
Ok, thank you for you help
